lib/Dialect/FIRRTL/NLATable.cpp

```diff
@@ -52,6 +73,28 @@ void NLATable::renameModule(StringAttr oldModName, StringAttr newModName) {
   symToOp.erase(oldModName);
 }
 
+void NLATable::renameModuleAndInnerRef(
+    StringAttr newModName, StringAttr oldModName,
+    const DenseMap<StringAttr, StringAttr> &renameMap,
+    llvm::Optional<SmallVector<NonLocalAnchor> *> updatedNLAs) {
+  auto op = symToOp.find(oldModName);
+  if (op == symToOp.end())
+    return;
+  auto iter = nodeMap.find(oldModName);
+  if (iter == nodeMap.end())
+    return;
+  ;
+  for (auto nla : iter->second) {
+    if (newModName != oldModName)
```

Comment:
if the names are the same, isn't this whole function unnecessary?

Reply:
I was replicating the logic for dedup pass, refactored to code to avoid this.